transport-native-epoll/src/test/java/io/netty/channel/epoll/EpollSocketTcpMd5Test.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel.epoll;
+
+import io.netty.bootstrap.Bootstrap;
+import io.netty.channel.ChannelInboundHandlerAdapter;
+import io.netty.channel.ConnectTimeoutException;
+import io.netty.channel.EventLoopGroup;
+import io.netty.util.CharsetUtil;
+
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.util.Collections;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class EpollSocketTcpMd5Test  {
+    private static final byte[] SERVER_KEY = "abc".getBytes(CharsetUtil.US_ASCII);
+    private static final byte[] BAD_KEY = "def".getBytes(CharsetUtil.US_ASCII);
+    private EventLoopGroup group;
+    private EpollServerSocketChannel server;
+
+    @Before
+    public void setup() {
```

Comment:
Could we setup and destroy at `@BeforeClass` and `@AfterClass`? Shutting down an event loop group takes time.